05-clingen.Rmd

```diff
@@ -9,12 +9,14 @@ ottrpal::set_knitr_image_path()
 
 This chapter will summarize:
 
-- Key conclusions of the course
-- Additional learning resources
+- The ClinGen Organization
+- ClinGen Somatic Cancer Variant Curation Expert Panels
 
-## Key conclusions 
+## ClinGen Somatic 
 
+The ClinGen organization, which leverages expert working groups to tailor guidelines for specific genes and provide public curated variant interpretations, has similarly developed a parallel effort in the somatic space. Multiple ClinGen Somatic Cancer Variant Curation Expert Panels (SC-VCEPs) are being developed around different genes or disease groupings (FGFR, NTRK, etc), and undergoing a four step process for approval. SC-VCEP somatic curation follows data models which incorporate the above mentioned AMP/ASCO/CAP and ClinGen/CGC/VICC guidelines for clinical relevance and oncogenicity, which may be tailored for specific genes and disease contexts.The CIViC knowledgebase, discussed below, has been selected as the primary database and curation interface for SC-VCEP variant classifications.
```

Comment:
Since this is its own chapter, it might be nice to clarify which effort this is similarly developed in parallel to.